client/src/lib/dependency-injection/graphql/buildUpdateMutation.js

```diff
@@ -1,15 +1,9 @@
 import { defaultTag } from './tags';
-import { getSingularName, getVariables, getParams, getFields, getFragments } from './helpers';
+import { getSingularName, getMutationVariables, getMutationParams, getFields, getFragments } from './helpers';
 
 const buildUpdateMutation = (tag = defaultTag) => (
-  tag`mutation Update${getSingularName}(
-    $Input:${getSingularName}UpdateInputType!
-    ${getVariables}
-  ) {
-    update${getSingularName}(
-      Input: $Input
-      ${getParams}
-    ) {
+  tag`mutation Update${getSingularName}${getMutationVariables} {
+    update${getSingularName}${getMutationParams} {
       ${getFields}
```

Comment:
Turning `$Input:${getSingularName}UpdateInputType!` and `Input: $Input` into default fallbacks, which can be overridden by setting explicit params. See the new related test for an example.